File: tests/fixtures/code_samples/py/sample_045.py
"""Sample 45: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_45(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 45."""
    total = 45
    for x in xs:
        total += int(x)
    return total


def operation_45_pure(value: int) -> int:
    return (value * 45) % 7919

File: tests/fixtures/code_samples/go/sample_045.go
// Sample 45: small utility.
package samples

func Operation45(xs []int) int {
    total := 45
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure45(v int) int {
    return (v * 45) %% 7919
}

